acra-censor/acra-censor_implementation.go

```diff
@@ -110,3 +103,39 @@ func (acraCensor *AcraCensor) HandleQuery(rawQuery string) error {
 	acraCensor.logger.Infof("Allowed query: '%s'", queryWithHiddenValues)
 	return nil
 }
+
+// GetLoggingTimeout returns current timeout of censor's logging process
+func (acraCensor *AcraCensor) GetLoggingTimeout() time.Duration {
+	return acraCensor.parsedQueriesWriter.GetSerializationTimeout()
+}
+
+// SetLoggingTimeout sets timeout of censor's logging process
+func (acraCensor *AcraCensor) SetLoggingTimeout(duration time.Duration) {
+	acraCensor.parsedQueriesWriter.SetSerializationTimeout(duration)
+	acraCensor.unparsedQueriesWriter.SetSerializationTimeout(duration)
+}
+
+func (acraCensor *AcraCensor) saveUnparsedQuery(query string) {
+	if acraCensor.unparsedQueriesWriter != nil {
+		saveQuery(acraCensor.unparsedQueriesWriter, query)
```

Comment:
imho better to use `unparsedQueriesWriter.WriteQuery(query)` or move `saveQuery` to file with this handler